components/advanced_create_comment/advanced_create_comment.test.tsx

```diff
@@ -0,0 +1,1773 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Did you mean to submit this file with this PR? It seems unrelated to the ticket

Reply:
It's also what's causing the lint and type-check CI jobs to fail as well

Reply:
My bad, somehow I messed up my branches and didn't see the file coming through commit. Fixing it right now.

Reply:
No worries!